doc/changelog.rst

```diff
@@ -3,8 +3,156 @@
 Changelog
 =========
 
+.. _changelog-v2-11-0:
+
+v2.11.0
+-------
+Highlights
+
+TODO: add highlights
+
+This release includes
+
+Backwards incompatible changes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+TODO: add examples of how to deal with these changes
+
+-  Allow contiguous representation of extracted regions (:pull:`2230`) by :user:`rebeccaherman1`
+
+Deprecations
+~~~~~~~~~~~~
+
+-  Refactor regridding (:pull:`2231`) by :user:`schlunma`
+-  Allow deprecated regridding scheme `linear_extrapolate` in recipe checks (:pull:`2324`) by :user:`schlunma`
```

Comment:
I think this is part of the same work as the pull request above, so i would make it a single entry